packages/good-design/src/hooks/useFVModalAction.ts

```diff
@@ -21,9 +21,12 @@ export const useFVModalAction = ({ firstName, method, redirectUrl = defaultRedir
       await fvlink?.getLoginSig();
       await fvlink?.getFvSig();
     } catch (e: any) {
-      setLoading(false);
       return;
+    } finally {
+      setLoading(false);
     }
+    
+    onClose();
```

Comment:
why do we need to close modal specifically?

Reply:
then remove it from props, totally, and from claim button which passes onClose callback

Reply:
but you removed it just partially I thought it's mistake